Packaging/fedora/devilutionx.spec

```diff
@@ -0,0 +1,50 @@
+%define debug_package %{nil}
+
+Name:		devilutionx
+Version:	0.3.1
```

Comment:
can we use %{version} here?

Reply:
You mean the first line?
It's there to suppress a possible build of a debug package.

Reply:
No, instead of having 0.3.1 hardcoded

Reply:
%{version} is basically defined by "Version:	0.3.1".
I need to hardcode it at least once. Otherwise the packages won't get a proper naming-scheme.

Reply:
Ok, was hoping it was able to read it from the repo 🙂